cluster-install/install.sh

```diff
@@ -2067,6 +2271,27 @@ EOF
     logdebug "$kustomization_file: added namespace transformer ($transformer_file_name)"
 }
 
+step_kustomize_global_pull_secret_if_needed() {
```

Comment:
Turns out we weren't adding the IMAGE_PULL_SECRET to fresh installs of the Trident or Connector operators. Didn't catch it since those are open source, but would've been a problem for dark sites.